samples/Calling/src/app/views/HomeScreen.tsx

```diff
@@ -103,3 +144,26 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
     </Stack>
   );
 };
+
+/**
+ * splits the participant Id's so we can call multiple people.
+ *
+ * will also make it raw id format for phone numbers
+ */
+const parseParticipants = (participantsString: string | undefined): string[] | undefined => {
+  console.log(participantsString);
+  if (participantsString) {
+    const participants = participantsString.split(', ');
```

Comment:
This is very flaky logic. This will fail if someone forgets to add a space after a comma.
I would sanitize the input first (removing any spaces) and then try to parse it.